src/components/Navigation.css

```diff
@@ -0,0 +1,98 @@
+.nav-container {
+  width: 100%;
+  height: 92px;
+  background-color: rgb(236, 236, 245);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  box-shadow: 2px 2px 12px rgb(109, 107, 107);
+}
+
```

Comment:
- [ ] Well done styling the app however you should follow the Zeplin design (Same fonts, colors, spacing) and its not the case for the Navbar. Kindly change the colors the font and spacing to match the design.